ui/src/app/app-routing.tsx

```diff
@@ -23,26 +26,34 @@ const CookiePolicyPage = withRouteData(CookiePolicy);
 const DataUserCodeOfConductPage = fp.flow(withRouteData, withFullHeight)(DataUserCodeOfConduct);
 const SessionExpiredPage = withRouteData(SessionExpired);
 const SignInAgainPage = withRouteData(SignInAgain);
+const SignInPage = withRouteData(SignIn);
 const UserAuditPage = withRouteData(UserAudit);
 const UserDisabledPage = withRouteData(UserDisabled);
 const WorkspaceAuditPage = withRouteData(WorkspaceAudit);
 
-export const AppRoutingComponent: React.FunctionComponent<{signIn: Function}> = ({signIn}) => {
+interface RoutingProps {
```

Comment:
IIRC we're going to end up accumulating a few more of these bound Angular type functions until we delete all the Angular so here is a place to stick them all instead of writing them out a few thousand times